### PR TITLE
Adding citation file

### PR DESCRIPTION
**Pull request by gvwilson**
Adding a CITATION.cff file

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,50 @@
+cff-version: 1.2.0
+message: "Please cite this paper as shown."
+authors:
+- family-names: "Wilson"
+  given-names: "Greg"
+  orcid: "0000-0001-8659-8979"
+- family-names: "Bryan"
+  given-names: "Jennifer"
+  orcid: ""
```
Comment (jennybc) [inline CITATION.cff]:
```suggestion
  orcid: "0000-0002-6983-2759"
```

Thanks for doing this!

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: "Good enough practices in scientific computing"
+preferred-citation:
+  type: article
+  authors:
+  - family-names: "Wilson"
+    given-names: "Greg"
+    orcid: "0000-0001-8659-8979"
+  - family-names: "Bryan"
+    given-names: "Jennifer"
+    orcid: ""
```
Comment (jennybc) [inline CITATION.cff]:
```suggestion
    orcid: "0000-0002-6983-2759"
```

**Comment by jkitzes** [comment]
@gvwilson Thanks for working on this - my ORCID is 0000-0001-7839-3594

**Comment by lexnederbragt** [comment]
Thanks, Greg! My ORCID is 0000-0001-5539-0999.

**Pull request closed without merging**